### ISSUE TITLE
[Bug]: Login loop

### ISSUE THREAD

**Issue by trparky** [ciderapp/Cider#1404]
### Describe the Bug

Cider keeps telling me that I have to log into my Apple account, over and over and over again. No matter what I do, it keeps asking me to login.

### Steps to Reproduce

1. Open Cider.
2. Attempt to login.
3. Watch Cider "spin its wheels".
4. Click the button in the upper left corner of the Cider window.
5. Cider asks me to login again.

### Environment Information

- OS: Windows 11
- App Version and/or Commit ID: 1.5.3

### Code of Conduct

- [X] I agree to follow this Cider's [Code of Conduct](https://github.com/ciderapp/Cider/blob/main/CODE_OF_CONDUCT.md)

### Disclaimer

- [X] I acknowledge the [Support Disclaimer](https://docs.cider.sh/support/disclaimer)

**Comment by BrunoBernardino** [comment]
I'm also running into this issue on Linux, started today. Version 1.5.1-beta.435

**Comment by rowbawts** [comment]
This is due to running a client older than 1.5.6. Update and it will be resolved.

**Comment by meechgalhuquot** [comment]
> This is due to running a client older than 1.5.6. Update and it will be resolved.

This version needs to be pushed as the Winget version since that still downloads 1.5.3

**Comment by BrunoBernardino** [comment]
I can confirm this was fixed after I updated to 1.5.6.

**Comment by trparky** [comment]
> > This is due to running a client older than 1.5.6. Update and it will be resolved.
> 
> This version needs to be pushed as the Winget version since that still downloads 1.5.3

Exactly. Until someone mentioned here, I didn't even know that there was a new version.

**Comment by cryptofyre** [comment]
> > > This is due to running a client older than 1.5.6. Update and it will be resolved.
> > 
> > 
> > This version needs to be pushed as the Winget version since that still downloads 1.5.3
> 
> Exactly. Until someone mentioned here, I didn't even know that there was a new version.

We announced it both on Twitter twice and Discord twice.

**Comment by trparky** [comment]
> > > > This is due to running a client older than 1.5.6. Update and it will be resolved.
> > > 
> > > 
> > > This version needs to be pushed as the Winget version since that still downloads 1.5.3
> > 
> > 
> > Exactly. Until someone mentioned here, I didn't even know that there was a new version.
> 
> We announced it both on Twitter twice and Discord twice.

Youy have a Twitter feed?

**Comment by cryptofyre** [comment]
> > > > > This is due to running a client older than 1.5.6. Update and it will be resolved.
> > > > 
> > > > 
> > > > This version needs to be pushed as the Winget version since that still downloads 1.5.3
> > > 
> > > 
> > > Exactly. Until someone mentioned here, I didn't even know that there was a new version.
> > 
> > 
> > We announced it both on Twitter twice and Discord twice.
> 
> You have a Twitter feed?

Yes.
https://twitter.com/UseCider

**Comment by trparky** [comment]
> > > > > > This is due to running a client older than 1.5.6. Update and it will be resolved.
> > > > > 
> > > > > 
> > > > > This version needs to be pushed as the Winget version since that still downloads 1.5.3
> > > > 
> > > > 
> > > > Exactly. Until someone mentioned here, I didn't even know that there was a new version.
> > > 
> > > 
> > > We announced it both on Twitter twice and Discord twice.
> > 
> > 
> > You have a Twitter feed?
> 
> Yes. https://twitter.com/UseCider

Shows you how behind the times I am. I didn't know that.

I would suggest that you guys get WinGet to know that there's an update.

I do have to ask though, what changed?

**Comment by yazninja** [comment]
> I would suggest that you guys get WinGet to know that there's an update.

Hi @trparky, Maintainer for Cider winget here, it has been updated there as well. It took a day to review there ( @meechgalhuquot tagging you for the notification )

**Comment by NovaViper** [comment]
Has this been updated for the Flatpak version? I'm having this issue currently on Linux Mint 21 using the Flatpak version of Cider

**Comment by MooCrumpus** [comment]
Still or again: login loop, archlinux AUR, V1.6.0.
